graphql/me/fields.ts

```diff
@@ -17,4 +19,21 @@ export class FieldMeResolver {
     myRoles(@Ctx() context: GraphQLContext): string[] {
         return context.user?.roles ?? [];
     }
+
+    // * pass role
+    @Query((returns) => String)
```

Comment:
@Jonasdoubleyou 
Is it enough to annotate it with `FieldResolver` or do you want me to put it in the session user?
With the FieldResolver Annotation it would be queryable like this:
```gql
query myZak {
  me {
    firstname
  }
  zoomZAK
  zoomSDKJWT(meetingId: "1", role: 1)
}
```

Reply:
I would find `me { zoomZak }` much better. Otherwise we sooner or later have many "loose queries" and not an interconnected graph